src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/_help.py

```diff
@@ -10,4 +10,12 @@
 helps['configure'] = """
             type: command
             short-summary: Configure Azure CLI 2.0 Preview or view your configuration. The command is interactive, so just type `az configure` and respond to the prompts.
+            examples:
+                - name: configure a default resource group
+                  text: >
+                    az configure --setting resource_group myRG
+                - name: configure a default webapp
+                  text: >
+                    az configure --section compute/vm --value myvm
```

Comment:
Having just 'vm' would mean it gets put in 'core':
```
[core]
vm = myvm
```
IMO that doesn't look quite right as it shouldn't be under core.

Suggestion, 'vm/name':
```
[vm]
name = myvm
```
`export AZURE_VM_NAME=myvm`